lib/pbench/server/api/__init__.py

```diff
@@ -70,6 +72,55 @@ def create_app():
     return app
 
 
+def _build_cors_preflight_response():
+    response = make_response()
+    response.headers.add("Access-Control-Allow-Origin", "*")
+    response.headers.add("Access-Control-Allow-Headers", "*")
+    response.headers.add("Access-Control-Allow-Methods", "*")
+    return response
+
+
+def _corsify_actual_response(response):
+    response.headers.add("Access-Control-Allow-Origin", "*")
+    return response
+
+
+class Download(Resource):
+    def post(self):
+        global app
+        elasticsearch = "http://elasticsearch.perf.lab.eng.bos.redhat.com:9280"
```

Comment:
This needs to be read from the configuration file.  See https://github.com/distributed-system-analysis/pbench/blob/master/server/lib/config/pbench-server.cfg.example#L33.